### PR TITLE
framework version to 3.2.0, entity-devkit version to released 3.0.0

### PR DESCRIPTION
**Pull request by tcrossrfid**
### What changes were proposed in this pull request?

Update framework version to 3.2.0.

### How is this patch documented?

one line in pom.xml

### How was this patch tested?

Existing unit tests.

#### Depends On

Nothing.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -65,7 +65,7 @@
         <dependency>
             <groupId>net.smartcosmos</groupId>
             <artifactId>smartcosmos-user-entity-devkit</artifactId>
-            <version>3.0.1-SNAPSHOT</version>
+            <version>3.0.0</version>
```
Comment (andreassiegel) [inline pom.xml:68]: Why did you decrement the version here?

Comment (tcrossrfid) [reply pom.xml:68]: 3.0.1-SNAPSHOT not available from nexus.

Comment (andreassiegel) [reply pom.xml:68]: It should be: https://github.com/SMARTRACTECHNOLOGY/smartcosmos-user-entity-devkit/blob/master/pom.xml#L17

**Comment by andreassiegel** [comment]
This pull request is closed because it's actually behind `master`.

**Pull request closed without merging**